modules/render.js

```diff
@@ -1,16 +0,0 @@
-/* @flow */
-
-import { Component } from 'react';
-import { AppRegistry } from 'react-native';
-
-// eslint/eslint/issues/5211
-/* eslint-disable space-infix-ops */
-export const render = (component: ReactElement, appKey: string = 'App'): void => {
```

Comment:
Can we keep this as an Easter egg? I prefer this style over AppRegistry and use it in my apps. 👍